ansible/roles/operator-pipeline/templates/openshift/pipelines/community-release-pipeline.yml

```diff
@@ -119,6 +172,116 @@ spec:
         - name: lease-name
           value: "community-pipeline"
 
+    # call IIB to add the bundle to index
+    - name: add-bundle-to-index
+      runAfter:
+        - get-supported-versions
+      taskRef:
+        name: add-bundle-to-index
+      params:
+        - name: pipeline_image
+          value: "$(params.pipeline_image)"
+        - name: index_images
+          value: "$(tasks.get-supported-versions.results.indices)"
+        - name: bundle_pullspec
+          value: "$(params.registry)/$(params.image_namespace)/$(tasks.detect-changes.results.added_operator):$(tasks.detect-changes.results.added_bundle)-$(params.git_head_sha)"
+        - name: iib_url
+          value: "$(tasks.set-env-community.results.iib_url)"
+        - name: environment
+          value: "$(params.env)"
+        - name: kerberos_keytab_secret_name
+          value: "$(params.kerberos_keytab_secret_name)"
+        - name: kerberos_keytab_secret_key
+          value: "$(params.kerberos_keytab_secret_key)"
+        - name: index-image-destination
```

Comment:
You can remove these 2 parameters because they are only relevant to the hosted pipeline.